### PR TITLE
Bilikisu Blog post on her experience in the champions program

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

**Comment by yabellini** [comment]
Hi @steffilazerte this is ready for you to review. Thanks!

File: content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+palmerpenguins::penguins |> 
+    count(species) |> 
+    set_names(c("species", "count")) |> 
+    chartkickR::chartkickR(type = "PieChart", x=species, y= count,
+                          colors = list("#4f86f7", "#fc5a8d","yellow"))
+
+```
+
+{{< figure src = "bar-plot.png" width = "600" alt = "Bar Plot example" class = "center">}}
```
Comment (steffilazerte) [inline content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md:53]:
```suggestion
{{< figure src = "bar-plot.png" width = "500" alt = "Pie Chart example - Showing the breakdown by species with a hover tooltip reading 124 Gentoo" class = "center">}}
```

File: content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+```r
+
+bar_series_data <- penguins |> 
+  group_by(year) |> 
+  count(species)
+
+chartkickR(data=bar_series_data, x=year, y=n, 
+          group=species, type = "BarChart",curve = TRUE)
+```
```
Comment (steffilazerte) [inline content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md:66]:
```suggestion
```r
p_data <- penguins |> 
  group_by(year) |> 
  count(species)

chartkickR(data = p_data, 
                  x = year, y = n, 
                  type = "BarChart",
                  group = species, curve = TRUE)
```
```

File: content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+
+{{< figure src = "serie-plot.png" width = "600" alt = "Multiple Series Bar Plot example" class = "center">}}
```
Comment (steffilazerte) [inline content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md:69]:
```suggestion
{{< figure src = "serie-plot.png" width = "500" alt = "Multiple Series Bar Plot example - Showing a series of horizontal bars depicting the numbers of each species in each year" class = "center">}}
```

File: content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+
+One of the most rewarding aspects of the rOpenSci champion program was the collaboration with a diverse and supportive community led by Yanina Bellini Saibene. Through discussions, feedback, and shared insights, [ChartkickR](https://github.com/BWOlatunji/chartkickR) evolved into a tool that not only met my initial vision but also exceeded it with the input of fellow rOpenSci champions and mentors.
+
+
+The [rOpenSci champion program](/champions/) extends beyond coding; it's a holistic experience that fosters personal and professional growth. From honing technical skills to friendliness within the open-source community, the program equipped me with invaluable tools that will undoubtedly shape my future endeavors.
+
+
+In that sense, I already organized two events with my local R User Groups - [Abuja RUG](https://www.meetup.com/abuja-r-user-group-meetup/events/294198946/) and [R-Ladies Abuja](https://www.meetup.com/rladies-abuja/events/294198849/) Chapter - showcasing what I learned about building htmlwidget in R. Looking into the future, I will continue to improve on my project and share it with the rOpensci community.
```
Comment (steffilazerte) [inline content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md:84]:
```suggestion
In this end, I have already organized two events with my local R User Groups - [Abuja RUG](https://www.meetup.com/abuja-r-user-group-meetup/events/294198946/) and [R-Ladies Abuja](https://www.meetup.com/rladies-abuja/events/294198849/) Chapter - showcasing what I learned about building htmlwidget in R. Looking into the future, I will continue to improve on my project and share it with the rOpenSci community.
```

File: content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+
+My journey as a rOpenSci champion has been a rollercoaster of discovery, and ChartkickR stands as a reward. I am grateful for this opportunity given to me by rOpensci, the support of the rOpenSci community, the guidance of my mentor, and the opportunities for growth that this program has provided. As I look back on this experience, I am not only proud of the htmlwidget I created but also of the person I've become throughout this incredible journey.
```
Comment (steffilazerte) [inline content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md:87]:
```suggestion
My journey as a rOpenSci Champion has been a rollercoaster of discovery, and ChartkickR stands as a reward. I am grateful for this opportunity given to me by rOpenSci, the support of the rOpenSci community, the guidance of my mentor, and the opportunities for growth that this program has provided. As I look back on this experience, I am not only proud of the htmlwidget I created but also of the person I've become throughout this incredible journey.
```

File: content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+
+{{< figure src = "serie-plot.png" width = "600" alt = "Multiple Series Bar Plot example" class = "center">}}
+
+
+## Developing a new package as rOpenSci Champion
+
+
+Developing [ChartkickR ](https://github.com/BWOlatunji/chartkickR)was not without its challenges. I encountered hurdles in the coding process, faced moments of doubt, and navigated through the complexities of integrating R with JavaScript. However, each challenge was met with perseverance, and every triumph brought a sense of accomplishment that fueled my passion for the project. I would like to appreciate the support of [John Coene](https://www.linkedin.com/in/johncoene/) and the guidance from my mentor, [Christina Maimone.](https://ropensci.org/author/christina-maimone/)
+
```
Comment (steffilazerte) [inline content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md:76]: Do you have an example of a challenge you overcame that you could share in a couple of sentences?

**Review by steffilazerte — COMMENTED** [review]
Hi @yabellini, Hi @BWOlatunji!

This is a lovely note sharing your experiences with the Champion Program, thank you! I have suggested some minor edits to the text, and marked one place where perhaps you could add a few more details, if you would like to.
I also suggest some code formatting changes to make the code examples a bit more consistent which can be really helpful for readers less familiar with R.

Congratulations on your package, that's such an awesome achievement!

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [ ] publication date is ok -> Probably push back to Tuesday or Thursday?
* [X] alternative text of images is informative
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML field 'preface' is present if necessary

**Comment by yabellini** [comment]
We can change the date and move forward.

File: content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md (hunk 6 above)
Comment (yabellini) [reply content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md:76]: For example, I had to learn JavaScript to create the package in R.

Comment (yabellini) [inline content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md:76]:
```suggestion
For example, I had to learn JavaScript to create the package in R.
```

File: content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+  count(species)
+
+chartkickR(data = p_data, 
+                  x = year, y = n, 
+                  type = "BarChart",
+                  group = species, curve = TRUE)
+
+
+{{< figure src = "serie-plot.png" width = "500" alt = "Multiple Series Bar Plot example - Showing a series of horizontal bars depicting the numbers of each species in each year" class = "center">}}
+
```
Comment (yabellini) [inline content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md:73]:
```suggestion
```
```

File: content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+```r
+p_data <- penguins |> 
+  group_by(year) |> 
+  count(species)
+
+chartkickR(data = p_data, 
+                  x = year, y = n, 
+                  type = "BarChart",
+                  group = species, curve = TRUE)
+
```
Comment (yabellini) [inline content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md:70]:
```suggestion
```
```

File: content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+{{< figure src = "serie-plot.png" width = "500" alt = "Multiple Series Bar Plot example - Showing a series of horizontal bars depicting the numbers of each species in each year" class = "center">}}
+
+## Developing a new package as rOpenSci Champion
+
+
+Developing [ChartkickR ](https://github.com/BWOlatunji/chartkickR)was not without its challenges. I encountered hurdles in the coding process, faced moments of doubt, and navigated through the complexities of integrating R with JavaScript. However, each challenge was met with perseverance, and every triumph brought a sense of accomplishment that fuelled my passion for the project. I would like to appreciate the support of [John Coene](https://www.linkedin.com/in/johncoene/) and the guidance from my mentor, [Christina Maimone](/author/christina-maimone/).
```
Comment (yabellini) [inline content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md:76]:
```suggestion
Developing [ChartkickR ](https://github.com/BWOlatunji/chartkickR)was not without its challenges. I encountered hurdles in the coding process, faced moments of doubt, and navigated through the complexities of integrating R with JavaScript. However, each challenge, like having to learn JavaScript to understand the library and create the package in R, was met with perseverance, and every triumph brought a sense of accomplishment that fuelled my passion for the project. I would like to appreciate the support of [John Coene](https://www.linkedin.com/in/johncoene/) and the guidance from my mentor, [Christina Maimone](/author/christina-maimone/).
```

File: content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+
+As I went through several training sessions and mentoring meetings, I stumbled upon an opportunity to create something truly unique -- a htmlwidget[^1] that would simplify the process of creating interactive and visually appealing charts in R. This idea took root and blossomed into what is now ChartkickR.
+
+[^1]: By [definition](https://cran.r-project.org/web/packages/htmlwidgets/index.html), an R 'htmlwidget' is a framework for creating HTML widgets that render in various contexts including the R console, 'R Markdown' documents, and 'Shiny' web applications. 
+
+
+[ChartkickR](https://github.com/BWOlatunji/chartkickR), at its core, is an htmlwidget that seamlessly integrates with the [Chartkick JavaScript library](https://chartkick.com/). It provides R users with a straightforward interface to generate beautiful and interactive charts using R code. From line charts to pie charts, ChartkickR empowers users to visualize their data effortlessly using a single line of code.
+
+## Examples of ChartkickR code
+
```
Comment (yabellini) [inline content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md:26]:
```suggestion
The package has ten types of plots, and all are very simple to build; here are three classic examples using the Palmer Penguins package as data. 
```

File: content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+As I went through several training sessions and mentoring meetings, I stumbled upon an opportunity to create something truly unique -- a htmlwidget[^1] that would simplify the process of creating interactive and visually appealing charts in R. This idea took root and blossomed into what is now ChartkickR.
+
+[^1]: By [definition](https://cran.r-project.org/web/packages/htmlwidgets/index.html), an R 'htmlwidget' is a framework for creating HTML widgets that render in various contexts including the R console, 'R Markdown' documents, and 'Shiny' web applications. 
+
+
+[ChartkickR](https://github.com/BWOlatunji/chartkickR), at its core, is an htmlwidget that seamlessly integrates with the [Chartkick JavaScript library](https://chartkick.com/). It provides R users with a straightforward interface to generate beautiful and interactive charts using R code. From line charts to pie charts, ChartkickR empowers users to visualize their data effortlessly using a single line of code.
+
+## Examples of ChartkickR code
+The package has ten types of plots, and all are very simple to build; here are three classic examples using the Palmer Penguins package as data.
```
Comment (yabellini) [inline content/blog/2023-12-11-champions-program-2023-my-experience/index.en.md:26]:
```suggestion

The package has ten types of plots, and all are very simple to build; here are three classic examples using the Palmer Penguins package as data. 
```

**Pull request merged**